### PR TITLE
Build code from docs

### PR DESCRIPTION
**Pull request by Randy808**
The docs for this branch use the latest version of the code and this shouldn't be merged yet until the below tasks have been completed.

- [ ] Distribute latest release on crates.io and pypi
- [x] Remove extension traits that have methods we want to merge into gl-client
~~- [ ] Update docs for creds page~~
- [x] Add python examples
- [x] Add jobs to ensure both Rust and Python files are being validated (maybe we should validate Python files with mypy?)

### CODE REVIEW

**Review by ErikDeSmedt — COMMENTED** [review]
Question:
Why is there a `ca.crt`-file in `python/getting-started` but not in `rust/getting-started`?

**Review by nepet — COMMENTED** [review]
Very nice work @Randy808!

I just have some minor stuff. I'll add some thoughts about the credentials extensions to #412 directly

File: docs/src/getting-started/recover.md [hunk 1]
```diff
@@ -11,45 +11,12 @@ In order to recover access all you need to do is recover the `seed` from the BIP
...
-	scheduler = Scheduler(
-	  node_id=signer.node_id(),
-		network="bitcoin",
-		tls=tls,
-	)
-	res = scheduler.recover(signer)
+--8<-- "main.py:recover_node"
 	```
 
 Notice that we are using a `TlsConfig` that is not configured with a
```
[the remark this answers is not in the document]
Comment (nepet) [reply docs/src/getting-started/recover.md:22]: This is old stuff by the way that is in the lines below this comment

**Comment by cdecker** [comment]
@Randy808 how is this PR looking, can we merge it soon even if not quite perfect, we can always fix up later, as long as we don't forget about it.

**Comment by Randy808** [comment]
I didn't update the credentials doc but I can make that a separate issue. The only other thing on the checklist is publishing new versions of glclient

**Review by nepet — DISMISSED** [review]
ack 4ffebe627f34c18d11b52f9380a542655348048c

**Review by nepet — APPROVED** [review]

**Pull request merged**